website_text/for_authors.md

```diff
@@ -14,14 +14,26 @@ More details journal policies and procedures can be found in [our authors guide]
 
 # Before Submission
 
-Authors should first send a presubmission letter to an editor in the relevant area. 
+Authors **must** first send a presubmission letter to the section lead editor in the relevant area. 
 The contents of the  letter are described in the [Guide for writing articles for LiveCoMS](https://livecomsjournal.github.io/).
 **Submissions without an approved presubmission letter will be rejected without review, and authors will not receive any refund of their submission fee.**
 
 Once approved for submission, the authors should prepare the manuscript using [LaTeX](https://www.latex-project.org/) in a suitable public [GitHub](http://wwww.github.com) repository, using our [LaTeX template](https://github.com/livecomsjournal/article_templates). 
 Instructions on how to use the templates can be found [in the GitHub repository with the template files](http://https://github.com/livecomsjournal/article_templates). Submission for publication carries a $100 fee, the only fees associated with publication in LiveCoMS.
 
-A complete description of the submission process can be found in [the guide for writing an article for LiveCoMS](https://livecomsjournal.github.io/).
+A complete description of how to prepare an article can be found in [the guide for writing an article for LiveCoMS](https://livecomsjournal.github.io/).
+
+# Submission process
+
+Articles (after approval of the presubmission letter) are submitted on the journal site, from [the "For Authors" section](http://www.livecomsjournal.org/for-authors). When filling out the submission form, note the following instructions:
+
+* Even if a version of the article is posted on arXiv, do not select to import the submission information from arXiv.
+* Limit the abstract to 300 words, and should include the web address of the GitHub repository where the author's version of the article is hosted.
+* The main article manuscript should be of type `.pdf`.
+* As a Supporting File, please upload a image (at least 300 x 200 pixels, any standard image format, color preferred, less than 1MB in size). This image is important to choose carefully, as it is the image that will appear on the article's front page and whenever the paper is shared on social media. We suggest something evocative of the research, but containing less scientific detail than, say, the table of contents image for an ACS journal. 
+* If there are simulation input files or data files included as part of your article, they should be placed into a single compressed `.zip` directory, with a README file fully explaining how to use the, as "Supporting Files".  Data should be uploaded in nonproprietary formats (`.pdf` instead of `.ppt`/`.pptx`/`.key`, `.csv` instead of `.xls` or `.xlsx`). In most cases, explanatory information should be put directly in the article itself, as there are no page limits. Supporting Files such as movies can be uploaded separately.
```

Comment:
Again, prefer having these documents in the GitHub repo rather than attached to the submission; if necessary they can be attached here but we should push people towards using version-controlled, easily-updatable documents if at all possible.

(Note if they deposit stuff with us and then they want to, say, change it that will create headaches...)